engine/runtime/src/main/scala/org/enso/interpreter/instrument/command/EditFileCmd.scala

```diff
@@ -20,17 +22,21 @@ class EditFileCmd(request: Api.EditFileNotification) extends Command(None) {
     ctx: RuntimeContext,
     ec: ExecutionContext
   ): Future[Unit] = {
-    for {
-      _ <- Future { ctx.jobControlPlane.abortAllJobs() }
-      status <- ctx.jobProcessor.run(
-        EnsureCompiledJob(request.path, request.edits)
-      )
-      _ <-
-        if (status == EnsureCompiledJob.CompilationStatus.Success)
-          Future.traverse(executeJobs)(ctx.jobProcessor.run)
-        else
-          Future.successful(())
-    } yield ()
+    ctx.locking.acquireFileLock(request.path)
+    try {
+      ctx.executionService.getLogger
+        .log(Level.FINE, s"EditFileCmd ${request.path}")
```

Comment:
Is this logging new? If so, we should probably use the new logging infra.

Reply:
It's not new. Truffle provides it's own logging facilities (TruffleLogger), that should be used in runtime. Runtime logger is configured during the context creation in the language server.